tianshou/utils/optim.py

```diff
@@ -20,3 +23,42 @@ def optim_step(
     if max_grad_norm:
         nn.utils.clip_grad_norm_(module.parameters(), max_norm=max_grad_norm)
     optim.step()
+
+
+_STANDARD_TORCH_OPTIMIZERS = [
+    torch.optim.Adam,
+    torch.optim.SGD,
+    torch.optim.RMSprop,
+    torch.optim.Adadelta,
+    torch.optim.AdamW,
+    torch.optim.Adamax,
+    torch.optim.NAdam,
+    torch.optim.SparseAdam,
+    torch.optim.LBFGS,
+]
```

Comment:
Can we use their parent class instead of hardcoding things here? Otherwise, if pytorch has an upgrade that changes some of them, it would silently fail on our side.

Reply:
I purposedly didn't use the base class because the clone_optimizer method might not work for user-defined optimizers

Reply:
This already indicates that cloning the optimizer might not be the best approach in the first place.

Reply:
See convo below, resolving